readme.md

```diff
@@ -81,8 +85,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
-
+gerldo = Student.find_by(name: "Gerlado")   
+pbj = Sandwich.create(type: "Peanut Butter & Jelly", student_id: gerlado.id)
 ```
 
```

Comment:
:+1:

This works, although a more readable way to create the sandwich would be something like...

```ruby
pbj = geraldo.sandwiches.create( type: "Peanut Butter & Jelly" )
```